tests/search/vespa/test_vespa_concept_filters.py

```diff
@@ -58,29 +59,15 @@ def test_concept_filters(
         },
         expected_status_code=200,
     )
-    response: SearchResponse = SearchResponse.model_validate(response)
+    actual: SearchResponse = SearchResponse.model_validate(response)
 
-    assert len(response.families) > 0
-    for concept_filter in concept_filters:
-        for family in response.families:
-            for family_document in family.family_documents:
-                for passage_match in family_document.document_passage_matches:
-                    assert passage_match.concepts
-                    if concept_filter["name"] == "parent_concept_ids_flat":
-                        assert any(
-                            [
-                                concept_filter["value"]
-                                in concept.parent_concept_ids_flat
-                                for concept in passage_match.concepts
-                            ]
-                        )
-                    else:
-                        assert any(
-                            [
-                                concept.__getattribute__(concept_filter["name"])
-                                == concept_filter["value"]
-                                for concept in passage_match.concepts
-                            ]
-                        )
+    # Set arbitrary values, since we don't care about these changing.
+    #
+    # The snapshot testing library, syrupy, does provide other ways to achieve
+    # this, but I was struggling with getting it working at the time.
+    actual.total_time_ms = 0
+    actual.query_time_ms = 0
+
+    assert actual == snapshot()
```

Comment:
The kind of data we are using in these tests particularly lends itself to snapshots. 1. Because they are a large blob, and the change here really shows the benefits of using them, having reduced down those nested for loops into one line is really nice! 2. Because it really just speeds up how we've been handling fixtures anyway. In that we grab one copy of prod data and just check it looks like what we expect, and currently managing them when adding new fields is a faff.

Reply:
One concern that can be raised on snapshotting is that it can make it easier to miss issues. For example, if the tests are failing we might assume its just that the snapshots need updating. Do you have any thoughts on how we avoid just running the snapshot update in that situation? This could also happen with out current setup so its not at all a dealbreaker from my perspective. But would be helpful if there is something in place to mitigate even if its just a note in the readme on dealing with these